Servers/utils/vendorRisk.util.ts

```
@@ -113,3 +113,15 @@ export const deleteVendorRiskByIdQuery = async (
   );
   return result.rowCount !== null && result.rowCount > 0;
 };
+
+export const deleteVendorRisksForVendorQuery = async (
+  vendorId: number
+): Promise<Boolean> => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)